pyramid/scripts/pcreate.py

```diff
@@ -21,73 +22,78 @@ def main(argv=sys.argv, quiet=False):
 
 
 class PCreateCommand(object):
-    verbosity = 1 # required
-    description = "Render Pyramid scaffolding to an output directory"
-    usage = "usage: %prog [options] -s <scaffold> output_directory"
-    parser = optparse.OptionParser(usage, description=description)
-    parser.add_option('-s', '--scaffold',
-                      dest='scaffold_name',
-                      action='append',
-                      help=("Add a scaffold to the create process "
-                            "(multiple -s args accepted)"))
-    parser.add_option('-t', '--template',
-                      dest='scaffold_name',
-                      action='append',
-                      help=('A backwards compatibility alias for '
-                            '-s/--scaffold.  Add a scaffold to the '
-                            'create process (multiple -t args accepted)'))
-    parser.add_option('-l', '--list',
-                      dest='list',
-                      action='store_true',
-                      help="List all available scaffold names")
-    parser.add_option('--list-templates',
-                      dest='list',
-                      action='store_true',
-                      help=("A backwards compatibility alias for -l/--list.  "
-                            "List all available scaffold names."))
-    parser.add_option('--package-name',
-                      dest='package_name',
-                      action='store',
-                      type='string',
-                      help='Package name to use. The name provided is assumed '
-                           'to be a valid Python package name, and will not '
-                           'be validated. By default the package name is '
-                           'derived from the value of output_directory.')
-    parser.add_option('--simulate',
-                      dest='simulate',
-                      action='store_true',
-                      help='Simulate but do no work')
-    parser.add_option('--overwrite',
-                      dest='overwrite',
-                      action='store_true',
-                      help='Always overwrite')
-    parser.add_option('--interactive',
-                      dest='interactive',
-                      action='store_true',
-                      help='When a file would be overwritten, interrogate '
-                           '(this is the default, but you may specify it to '
-                           'override --overwrite)')
-    parser.add_option('--ignore-conflicting-name',
-                      dest='force_bad_name',
-                      action='store_true',
-                      default=False,
-                      help='Do create a project even if the chosen name '
-                           'is the name of an already existing / importable '
-                           'package.')
+    verbosity = 1  # required
+    parser = argparse.ArgumentParser(
+        prog="pcreate",
```

Comment:
`prog` is unnecessary... usually we'd leave this unspecified. For the docs you're already forcefully setting it via `:prog:` so that they will render with the selected name.